entry/.preview/cache/css.json

```diff
@@ -0,0 +1,5 @@
+{
+  "entry": {
```

Comment:
remove .preview folder, recheck and remove all unused folders / auto generated files